### PR TITLE
mobile Setup portfolio

### PR DESCRIPTION
**Pull request by sadat-Ramin**
I learn some css codes and solve the linters error.
Install some necessary things for project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        </li>
+        <li>
+            <a href="https://github.com/sadat-Ramin"><img src="image/icons/cat.svg" /></a>
+        </li>
+        <li>
+            <a href="#"><img src="image/icons/fingers.svg" /></a>
+        </li>
+    </ul>
+</div>
+</section>
```
Comment (Ofarouq310) [inline index.html:56]: - [ ] Kindly wrap your whole `section` with the `main` tag.

---

- Kindly take a look at the picture which displays the tags that have to be used inside the `body`:
  -  First of all the `header`, which in some cases includes the `nav` inside of it, like our portfolio, and in some cases it doesn't.
  - Secondly, which is not displayed in the picture, the `main` tag, which wraps everything related to your visible content (sections, articles, divs, images, etc...).
  - Last but not least, the `footer`.
  
- All in all, your `body` should consist of 3 tags: `header`, `main`, and `footer`.

    image

**Review by Ofarouq310 — CHANGES_REQUESTED** [review]
Hi @sadat-Ramin, ✋  

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥈

giphy

## Highlights
✅ Followed GitHub workflow.
✅ No linters errors.

## Required Changes ♻️

- [ ] Your Pull request title should be short, and concise, and addresses what you are implementing, in your case, it could be, `Add header and headline sections` or something similar, simpler. Please refer to the next picture to help you how to edit the title.
image

- [x] As for the PR description, you have to summarize what you implemented in the project using a bullet list, a checklist, or a numbers list, in your case it could be something like:
    - Added a header section to the portfolio,
    - Added a headline section with an introduction paragraph.
    - etc...
     
or anything that has the same intent of explaining the new additions to your project.


_Check the comments under the review._

## Optional suggestions

 - Concerning commit messages, kindly make sure they should be short, concise, descriptive of an action, written in the imperative tense, and capitalized, take a closer look in the next picture for some great descriptive examples.
image

- Your branch should have a name of a feature or action you are implementing, in your case it could be "mobile-setup" or "header-and-deadline" or whatever suits you. Kindly fix it and rename your branch. Kindly refer to the next image for the steps of renaming a branch.
image

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @sadat-Ramin, 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. You are almost there...



## Highlights

- Professional README.md ✅   
- Great attempt at design implementation ✅ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rica213** [comment]
Hi Sayed (@sadat-Ramin)
I suggest you continue requesting a review using this Pull Request instead of the [other one](https://github.com/sadat-Ramin/My-Portfolio/pull/2) you just created 🙏🏽 
That way, someone can claim your review ❤️ 

Kindly do that and have a good day 🌞

**Pull request closed without merging**